ui/src/main/resources/PhenoTips/SimilarCases.xml

```diff
@@ -1659,6 +1659,9 @@
     color: $theme.notificationErrorColor;
     font-size: .87em;
 }
+.match-details .variants-data a:visited {
+    color: #000000 !important;
```

Comment:
The UCSC links are implemented as "secondary action" buttons. You should use the color defined in the color theme.
```suggestion
    color: $theme.buttonSecondaryTextColor !important;
```